examples/diff.c

```diff
@@ -215,11 +227,14 @@ static int color_printer(
 static void parse_opts(struct opts *o, int argc, char *argv[])
 {
 	struct args_info args = ARGS_INFO_INIT;
+	const char **break_rewrite_arg;
```

Comment:
No because in the `optional_str_arg` needs first argument to be const char **, look [here](https://github.com/libgit2/libgit2/blob/77394a27af283b366fa8bb444d29670131bfa104/examples/common.h).